include/deal.II/grid/tria_accessor.h

```diff
@@ -3184,6 +3184,19 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
   TriaActiveIterator<DoFCellAccessor<dim, spacedim, false>>
   as_dof_handler_iterator(const DoFHandler<dim, spacedim> &dof_handler) const;
 
+  /**
+   * A function similar to as_dof_handler_iterator(). It converts
+   * a Triangulation active cell iterator to a
+   * DoFHandler active level iterator, or a DoFHandler active/level cell
+   * iterator
+   * to a level cell iterator of another DoFHandler. The @p iterator must be
```

Comment:
I do not understand the comment. What about
```suggestion
   * A function similar to as_dof_handler_iterator(). It converts
   * a Triangulation active/level cell iterator to a
   * DoFHandler level cell iterator, or a DoFHandler active/level cell
   * iterator
   * to a level cell iterator of another DoFHandler. The @p iterator must be
```